resources/views/emails/contactUs.blade.php

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+<p>User email :{{$email}}</p>
+<p>Name: {{$name}}</p>
+<p>Subject: {{$subjectMessage}}</p>
+<p>Message:</p>
+<p>{{$messageText}}</p>
+
```

Comment:
the hmtl is invalid, body closing tag is missing.

Reply:
Html fixed.